.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -14,31 +14,35 @@ name: AGW Test LTE Integration With Make Containerized Build
 on:
   workflow_dispatch:
     inputs:
-      digest_c:
+      hash_c:
```

Comment:
should these all be the same (usually?) - I would remove the complexity if true

Reply:
Well, they are not required to be, but I guess that in reality they usually are. When this job is called in CI, the images related to this commit are tested, so they all share the same hash. And when run manually we usually don't care about the specific images, but rather some integ tests.